server/api/settings.py

```diff
@@ -68,11 +73,45 @@
     "api.healthcheck",
     "corsheaders",
     "rest_framework",
-    "rest_framework_jwt",
+    "rest_framework_simplejwt",
     "app",
-    'drf_yasg',
+    "drf_yasg",
 ]
 
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": (
+        "rest_framework_simplejwt.authentication.JWTAuthentication",
+        "rest_framework.authentication.SessionAuthentication",
+        "rest_framework.authentication.BasicAuthentication",
+    ),
+    "DEFAULT_PERMISSION_CLASSES": (
+        "rest_framework.permissions.IsAuthenticatedOrReadOnly",
+    ),
+}
+
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(hours=5),
+    # Short-term access token lifetime
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=7),
+    # Long-term refresh token lifetime
+    "ROTATE_REFRESH_TOKENS": True,
+    # Rotate refresh tokens
+    "BLACKLIST_AFTER_ROTATION": True,
+    # Blacklist old tokens after rotation
+    "ALGORITHM": "HS256",
+    # Signing algorithm
+    "SIGNING_KEY": SECRET_KEY,
+    # Secret key for signing tokens
+    "AUTH_HEADER_TYPES": ("Bearer",),
+    # Authentication header type
+    "AUTH_HEADER_NAME": "HTTP_AUTHORIZATION",
+    # Authentication header name
+    "USER_ID_FIELD": "user_id",
+    # User ID field
+    "USER_ID_CLAIM": "user_id",
+    # User ID claim in the token
```

Comment:
hm? minor nitpick - comments should be above the code they are for, I don't think the comments are for the code underneath them?